### ISSUE TITLE
Training config of *ConsistencyXVC-w/o loss* model

### ISSUE THREAD

**Issue by tarepan** [ConsistencyVC/ConsistencyVC-voive-conversion#18]
## Summary
In the experiment of the paper, is *ConsistencyXVC-w/o loss* model trained with segment=28/nb=108?  

## Current Status
In the paper, there is a description of *ConsistencyXVC-w/o loss* model training config as below.  

>  ConsistencyXVC-w/o loss, on the other hand, continues training with the same parameters without introducing speaker consistency loss up to 300k steps.

The phrase "***with the same parameters***" coud have two meanings.   

- Same as *ConsistencyXVC-w/o loss*'s first 100 steps (it means segment=28/nb=108)
- Same as *ConsistencyXVC*'s fine-tuning step, except for SCL (it means segment=75/nb=42)

## Question
In the experiment of the paper, is *ConsistencyXVC-w/o loss* model trained with segment=28/nb=108 for 0~300k steps?  
Or, is it trained with 28/108 for 0 ~ 100k steps + 75/42 for 100 ~ 300k steps?  

It is merely english interpretaion question, sorry for bothering you.

**Comment by ConsistencyVC** [comment]
Same as ConsistencyXVC-w/o loss's first 100 steps (it means segment=28/nb=108)
